core/arguments.js

```diff
@@ -0,0 +1,46 @@
+var consolec = require('./unsafe/console.js'),
+	options = [
+		{
+			long: '--debug',
+			short: '-d',
+			run: function() {
+				console.warn('Debug mode enabled.');
+				consolec.setDebug(true);
+			}
+		},
+		{
+			long: '--log',
+			short: '-l',
+			run: function() {
+				console.warn('Logging mode enabled.');
+				consolec.setLog(true);
+			}
+		},
+		{
+			long: '--timestamp',
+			short: '-t',
+			run: function() {
+				consolec.setTimestamp(true);
+			}
+		}
+	];
+
+exports.runArguments = function(args) {
+	for (var i = 0; i < args.length; i++) {
+		var arg = args[i],
+			pargs = options.filter(function(value) {
```

Comment:
You're trying to make them easier and lazier to use. 

It would be preferable if they were never used except in development in my opinion. In some ways I think I should even remove the short versions so that they are both inconvenient to use and do you know precisely what each one does. 

We could also potentially run into problems that have been encountered in GNU getopt (ls and tar in particular) in future if we expand/change the options. Letters can become positionally dependant because one takes an argument. Eg. tar -xvf Foo.tar cannot be changed to tar -fxv Foo.tar because the -f has an argument.